includes/view/User_view.php

```diff
@@ -350,16 +351,27 @@ function User_view_myshift(Shift $shift, $user_source, $its_me)
     }
 
     if ($shift->freeloaded) {
-        $myshift['duration'] = '<p class="text-danger">'
-            . sprintf('%.2f', -($shift->end->timestamp - $shift->start->timestamp) / 3600 * 2) . '&nbsp;h'
```

Comment:
The subtraction & doubling should be only removed for non-goodie-configs or it is pretty confusing, especially when combined with night shifts

Reply:
I like it this way more because minus the double length doesnt count for the sum underneath 
night shifts are also not shown with double length

Reply:
Imho for the goodie none case its fine to just show the min red and cross them.
For goodie / shirt enabled the shown hours imho should indicate the shirt score as the total "real" hours can be easily counted.

So I would suggest having the shirt score value displayed and the "real" hours in round brackets after it like `-4.8h (2.4h)` when the user has missed a shift etc., eventually with a `title="T-Shirt score: -4.8h, 2.4h worked"`.